.cirrus.star

```diff
@@ -0,0 +1,4 @@
+load("github.com/SonarSource/cirrus-modules@v2", "load_features")
```

Comment:
What is this used for actually?
I'm not familiar with this.

Reply:
Yes so this is the starlak entry point. It loads all the features we can use in the pipeline. from this file https://github.com/SonarSource/cirrus-modules/blob/master/lib.star for example accessing the vault or even a default `only_if` function/condition